lib/utils.pm

```diff
@@ -28,6 +28,7 @@ use Utils::Architectures qw(is_aarch64 is_ppc64le);
 use Utils::Systemd qw(systemctl disable_and_stop_service);
 use Utils::Backends 'has_ttys';
 use Mojo::UserAgent;
+use qam;
```

Comment:
Regarding travis failure, there is something wrong with it, I don't see any reason why it failed.
I can make it pass when I remove this line, but then test will fail http://10.100.12.155/tests/16733#step/openvswitch/5

Reply:
![img](https://paste.opensuse.org/8a0edcfd)